src/backend/utils/load/age_load.c

```diff
@@ -126,15 +147,21 @@ void insert_edge_simple(Oid graph_id, char* label_name, graphid edge_id, graphid
     values[2] = GRAPHID_GET_DATUM(end_id);
     values[3] = AGTYPE_P_GET_DATUM((edge_properties));
 
-    label_relation = heap_open(get_label_relation(label_name, graph_id), RowExclusiveLock);
-    tuple = heap_form_tuple(RelationGetDescr(label_relation), values, nulls);
+    label_relation = heap_open(get_label_relation(label_name,
+                                                  graph_id),
+                               RowExclusiveLock);
+
+    tuple = heap_form_tuple(RelationGetDescr(label_relation),
+                            values, nulls);
     heap_insert(label_relation, tuple,
                 GetCurrentCommandId(true), 0, NULL);
     heap_close(label_relation, RowExclusiveLock);
     CommandCounterIncrement();
 }
 
-void insert_vertex_simple(Oid graph_id, char* label_name, graphid vertex_id, agtype* vertex_properties) {
+void insert_vertex_simple(Oid graph_id, char* label_name,
+                          graphid vertex_id,
+                          agtype* vertex_properties) {
```

Comment:
Brace needs to be on its own line -

some_function_or_command(blah blah,...)
{